### ISSUE TITLE
[Bug]: Integrated Terminals Storage terminal not working.

### ISSUE THREAD

**Issue by TheDarkPreacher** [FTBTeam/FTB-Modpack-Issues#1060]
### Modpack

FTB Presents Direwolf20 1.18

### Modpack version

1.6.1

### Describe the bug

After the latest update, the Integrated Terminals storage terminal no longer displays items in inventories that are connected to the network, nor does it accept power in order to run a storage network.

### Steps to reproduce

1: Create Integrated Dynamics network with storage
2: Add Storage Terminal
3: give power to network
4: open storage terminal.

### Expected behaviour

Expected behaviour: Open storage terminal, be able to access items, autocrafting, and regular crafting.

### Screenshots

![image](https://user-images.githubusercontent.com/15058826/177020731-f2b74d27-a426-43ac-b441-346c4577f0f8.png)

### Additional information

In previous versions, the terminal was working, even without giving power to the network. Not sure if this is an issue with Integrated Dynamics or Integrated Terminals, or simply a config problem.

**Comment by TheonlyTazz** [comment]
![image](https://user-images.githubusercontent.com/3908847/177289842-76050009-8289-4572-8ac2-023c0dd5106f.png)
![image](https://user-images.githubusercontent.com/3908847/177289868-65e1239b-ff1e-4715-957b-ad7bfa477944.png)
Item Storage working as intended, use an Interface to hook up chests


![image](https://user-images.githubusercontent.com/3908847/177289960-adda1de5-0a1c-45bc-a2cc-5503f3f14cc4.png)
![image](https://user-images.githubusercontent.com/3908847/177289991-f1211fea-06ca-4bd6-b153-8942f86f8c20.png)
Energy Storage working as intended, hook it up directly with a cable

**Comment by Corekiller28** [comment]
Im having this exact issue on 1.18 Enigmatic 8, after a server crash while the storage interface was loaded no matter what group I put the storage in, or any time I recreate it I get that blank gray screen as it feels like a corrupted network.

**Comment by korncola** [comment]
Sam issue on the All the Mods 9 0.0.45 (MC 1.20.1).
Also googling this issue gives results on reddit without a solution.

**Comment by Jasems** [comment]
Having this same issue in Arcane Engineering. Minecraft 1.18.2. Did anyone else figure out a work-around? For me the storage terminal worked fine for a week or so, but now does the same as everyone else in this post.

**Comment by Paddy69** [comment]
Having the same issue on 1.20.1 Valhelsia 6. after a disconnect from the server i no longer was able to use any storage terminal. the network as a whole moved items correctly but could not use the terminal to do so manually. Tested on a single chest and would still not read to the terminal.
![image_2023-12-05_184054898](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/63801613/426d0036-2823-4f3c-bd68-11c5c313f83b)
![image_2023-12-05_184110155](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/63801613/19cfc27d-31a0-46b9-9353-f843a93ee940)

**Comment by haphazodus** [comment]
Was setting up auto crafting on the network. When I attached a crafting table to the crafting interface the game crashed. When I reloaded the game the  Integrated Terminals Storage terminal no longer shows items in it. The rest of the network is still running, items are being moved in and out of the system, but nothing shows on the screen. 
I have tried a repair client via curseforge but it did not fix this issue.
![image](https://github.com/user-attachments/assets/c37052da-e229-42a8-bf7f-9d82b08208e9)
![image](https://github.com/user-attachments/assets/97c31885-a0b4-4b5a-804a-cf956fdfa2e1)
